src/docs-app/routes.coffee

```
@@ -0,0 +1,31 @@
+angular.module("ThemisComponentsApp")
+  .config ($stateProvider, $locationProvider, $urlRouterProvider) ->
+    $locationProvider.html5Mode true
+
+    $stateProvider
+      .state
+        name: "index"
+        url: "/"
+        component: "docsComponentDetails"
+
+      .state
+        name: "component"
+        url: "/component/:name"
```

Should the `.state` function expect a string to be used here. We were having problems in Themis where the ui-router was trying to parse a `string` value into a id integer and failing hard. wondering if this could be a problem with ThemisUI if we were planning to add query values to the url.